www/class/centreonConfigCentreonBroker.php

```diff
@@ -958,8 +936,8 @@ public function getCorrelationFile()
         $row = $res->fetchRow();
         $configId = $row['config_id'];
         $correlationGroupId = $row['config_group_id'];
-        $query = 'SELECT config_value FROM cfg_centreonbroker_info
-            WHERE config_key = "file" AND config_id = ' . $configId . ' AND config_group_id = ' . $correlationGroupId;
+        $query = 'SELECT config_value FROM cfg_centreonbroker_info  WHERE config_key = "file" AND config_id = '
```

Comment:
for better readability, please send the WHERE clause on next line and the AND clauses on new lines